src/rpdk/core/data_loaders.py

```diff
@@ -148,6 +141,24 @@ def load_resource_spec(resource_spec_file):  # noqa: C901
         LOG.debug("Resource spec validation failed", exc_info=True)
         raise SpecValidationError(str(e)) from e
 
+    if (
+        "maxresults" in map(str.lower, resource_spec.get("properties", []))
+        or "nexttoken" in map(str.lower, resource_spec.get("properties", []))
+        or "nextmarker" in map(str.lower, resource_spec.get("properties", []))
+    ):
```

Comment:
I've also seen `Marker`, `ClientToken` and `StatusReason` all as problematic properties that are clearly not assignable.

Reply:
@iann0036 thanks for the heads up, I'll look into these tomorrow:
[`AWS::EFS::AccessPoint.ClientToken`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-efs-accesspoint.html#cfn-efs-accesspoint-clienttoken)
[`AWS::LicenseManager::License.ClientToken`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-licensemanager-license.html#cfn-licensemanager-license-clienttoken)
[`AWS::LicenseManager::Grant.ClientToken`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-licensemanager-grant.html#cfn-licensemanager-grant-clienttoken)
[`AWS::LicenseManager::Grant.StatusReason`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-licensemanager-grant.html#cfn-licensemanager-grant-statusreason)

Reply:
was easy to track down LIST API option synonyms using paginators, but left those open while we try to find a way to track down all their synonyms as well